fastseq/optimizer/fairseq/el_attention_optimizer.py

```diff
@@ -457,7 +458,7 @@ def extract_features(
         if self.project_out_dim is not None:
             x = self.project_out_dim(x)
 
-        return x, {'attn': attn, 'inner_states': inner_states}
+        return x, {"attn": [attn], "inner_states": inner_states}
```

Comment:
Curious: why using `"` instead of `'`?

Reply:
That's how it is in [fairseq v0.10.2](https://github.com/pytorch/fairseq/blob/83e615d66905b8ca7483122a37da1a85f13f4b8e/fairseq/models/transformer.py#L829). Not sure why it changed between 0.9.0 and 0.10.2.